felix/dataplane/linux/int_dataplane.go

```diff
@@ -423,12 +439,22 @@ func NewIntDataplaneDriver(config Config) *InternalDataplane {
 		OnStillAlive:          dp.reportHealth,
 		OpRecorder:            dp.loopSummarizer,
 	}
+	nftablesOptions := nftables.TableOptions{
```

Comment:
Need to evaluate this TableOptions struct to determine which are relevant for nftables. For now, I've mostly just copied the iptables implementation.